scipy/stats/_qmc.py

```diff
@@ -347,7 +388,7 @@ def update_discrepancy(x_new, sample, initial_disc):
     return _cy_wrapper_update_discrepancy(x_new, sample, initial_disc)
 
 
-def primes_from_2_to(n):
+def primes_from_2_to(n: IntNumber) -> np.ndarray:
```

Comment:
This function is private and only called by `n_primes` using `int`.
```suggestion
def primes_from_2_to(n: int) -> np.ndarray:
```